### PR TITLE
Remade the countdown timer to be more relevant and fitting

### PR DESCRIPTION
**Pull request by Illnoid**
Change countdown timer from when food is delivered to food is ordered.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '2.2'
+version: '2.0'
```
Comment (Dr-Horv) [inline docker-compose.yml]: Downgrade?

File: pizza.html [hunk 2]
```diff
@@ -100,7 +98,7 @@ <h3>🍽 Place your order</h3>
 	{{#if timer_started}}
     <div class="container-part">
         <div class="container-content">
-            <h3 class="text-extra-style">🕑 Time until food's ready:</h3>
+            <h3 class="text-extra-style">🕑 Time until food's ordered:</h3>
```
Comment (Dr-Horv) [inline pizza.html]: So this PR changes from time until ready to time until order?

File: pizza.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 OrderItems = new Mongo.Collection("order_items");
 Orders = new Mongo.Collection("orders");
+var timeToOrder = false;
```
Comment (Dr-Horv) [inline pizza.js]: Variable name is a bit confusing, I thought it was a duration (10 minutes or so). Should perhaps be something like, hasOrdered or something similar indicating the boolean nature

File: pizza.js [hunk 4]
```diff
@@ -100,10 +101,12 @@ if (Meteor.isClient) {
       $('.new-pizza [name="pizza"]').val(name).next().focus();
     },
     "click .pizza-item small.deletable": function(event) {
-      if (confirm("Are you sure?")) {
-        var id = $(event.target).data('id');
-        OrderItems.remove({_id: id});
-      }
+      if (timeToOrder === false) {
```
Comment (Dr-Horv) [inline pizza.js]: !timeToOrder

Comment (Dr-Horv) [reply pizza.js]: Actually

`!timeToOrder && confirm("Are you sure?")`

File: pizza.js [hunk 5]
```diff
@@ -153,7 +157,11 @@ if (Meteor.isClient) {
   }
   Template.pizza.helpers({
     classname: function() {
-      return timer_started() ? '' : 'deletable editable';
+	if(timeToOrder === false) {
```
Comment (Dr-Horv) [inline pizza.js]: `return timeToOrder ? '' : 'deletable editable'`

Alternatively
```
if(!timeToOrder) { 
   return 'deletable editable'
} else {
   return ''
}

**Comment by Portals** [comment]
Because we're working on a ReactJS version of EatIT, we'll have to unfortunately close this PR.

**Pull request closed without merging**